src/shared/model/experiments/tests/passcode-signin-test.ts

```diff
@@ -0,0 +1,25 @@
+import { ABTest } from '@guardian/ab-core';
+
+export const passcodeSignInTest: ABTest = {
+	id: 'PasscodeSignInTest', // This ID must match the Server Side AB Test
+	start: '2025-01-13',
+	expiry: '2025-01-31', // Remember that the server side test expiry can be different
+	author: 'mahesh.makani@theguardian.com',
+	description:
+		'Testing the release of one time passcodes as the default sign in option',
+	audience: 0.1, // 10% (1 is 100%)
```

Comment:
So does this mean that once this PR is merged 10% of users will start seeing the new UX or is there something else that has to be toggled first?

Reply:
Yeah, it will go out to 10% of users when the PR is merged in! The switch is already set to on.